shorebird_code_push/example/lib/main.dart

```diff
@@ -132,7 +132,13 @@ class _MyHomePageState extends State<MyHomePage> {
 
   Future<void> _downloadUpdate() async {
     _showDownloadingBanner();
-    await _shorebirdCodePush.downloadUpdateIfAvailable();
+
+    await Future.wait([
+      _shorebirdCodePush.downloadUpdateIfAvailable(),
+      // Add an artificial delay so the banner has enough time to animate in.
+      Future<void>.delayed(const Duration(seconds: 1)),
```

Comment:
minor but looks like it doesn't need to be a full second:
https://github.com/flutter/flutter/blob/b71f8ce7ae195ab76c69a3a25f569d9dca15e20f/packages/flutter/lib/src/material/banner.dart#L18

Reply:
Done.